### ISSUE TITLE
Dice Loss

### ISSUE THREAD

**Issue by postBG** [postBG/BodyPartsSegmentation.pytorch#13]

**Comment by postBG** [comment]
https://github.com/keras-team/keras/issues/9395

**Comment by postBG** [comment]
https://arxiv.org/pdf/1801.05912.pdf

**Comment by postBG** [comment]
Fuck this loss
